.ergomake/docker-compose.yaml

```
@@ -0,0 +1,38 @@
+version: "3"
+services:
+  app:
+    build: ../
+    ports:
+      - "8080:8080"
+    environment:
+      BASE_URL: https://{{ services.api.url }}
+      AUTH_SECRET: MdVsXMQs6B2CmKx7u400Etrog3xYpcpN
+      DATABASE_URL: Server=postgres;Port=5432;User Id=aptabase;Password=aptabase_pw;Database=aptabase
+      CLICKHOUSE_URL: Host=clickhouse;Port=8123;Username=aptabase;Password=aptabase_pw
+      ASPNETCORE_URLS: http://+:8080
+      SMTP_HOST: mailhog
+      SMTP_PORT: 1025
```

Then you add the `replace-env` label here, so ergomake replaces your env var evaluating the template and making it have the dynamic value.
```suggestion
      SMTP_PORT: 1025
    labels:
      dev.ergomake.env.replace-env.BASE_URL: "https://{{ services.app.url }}"
```